controller/engine_controller.go

```diff
@@ -1504,6 +1505,11 @@ func handleRestoreError(log logrus.FieldLogger, engine *longhorn.Engine, rsMap m
 	return nil
 }
 
+func checkFailedLock(testString string) bool {
```

Comment:
Rename to `isReplicaRestoreFailedLockError(err *client.ReplicaError)`

Reply:
Modified. Thanks.